mne/beamformer/tests/test_rap_music.py

```diff
@@ -47,11 +48,12 @@ def _get_data(event_id=1):
                                                surf_ori=True)
 
     noise_cov = mne.read_cov(fname_cov)
+    noise_cov = regularize(noise_cov, evoked.info)
```

Comment:
Not necessary for the test, but the evoked.plot_white was having the same problem I had before.
Should I remove it?